ru.json

```diff
@@ -239,5 +239,14 @@
         "Universe": "Галактика",
         "GOLDEN WIN": "Всё или ничего",
         "Crystal Karambit": "Кристалический кейс"
+        "Knives" : " Ножевые кейсы"
```

Comment:
Thanks again for your awesome work. 🤓
For future changes it would be create when you make sure to add a comma (`,`) after each end of a line.